druval_cr/postgresAssignment/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "college_application",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "ts-node-dev src/server.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@koa/router": "^10.0.0",
+    "koa": "^2.13.1",
+    "koa-bodyparser": "^4.3.0",
+    "koa-combine-routers": "^4.0.2",
+    "pg": "^8.6.0",
```

Comment:
we can use https://sequelize.org/master/ for db operation and https://joi.dev/api/?v=17.4.0 for request validation. Try it out in simple project

Reply:
added joi validation